### DISCUSSION TITLE
Problem with CSS in Version alpha73

### DISCUSSION THREAD

**Discussion by pedrohcmds** [facebook/docusaurus#4684] [Q&A]
Hello, 

Lately i have upgrade my docussaurus site to version alpha 73, and it broke some CSS rules for different pages. My last example was the most intrigant:

`
.exMembro{
  display: inline-grid;
  text-align: center;
  width: 33%;
  padding: 2rem;
  margin: auto;
}`

This was the code,  but when i make a build, the display were overwritten to block, as it have ignored the rules.
After this, I added !important to the CSS rule and it worked. Is there any better form to do that? Is there some modification in the css hierarchy?

**Comment by slorber** [comment]
It's impossible to help you without a repro. 

`. exMembro` is not a Docusaurus class we define, we can't know why it use block or grid
